Components/Steps.js

```diff
@@ -16,7 +16,6 @@ const Steps = () => {
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
```

Comment:
I had to remove this flex-grow setting to be able to render both the steps and sprite component in the same component